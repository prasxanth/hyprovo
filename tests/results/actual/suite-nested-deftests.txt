2021-11-24 13:52:30.376 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results): add pass
2021-11-24 13:52:30.378 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results): sub fail 
                                    Expected: 2
                                    Got: -1
2021-11-24 13:52:30.383 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-sub): sub pass
2021-11-24 13:52:30.385 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-sub): sub fail 
                                    Expected: 4
                                    Got: -1
2021-11-24 13:52:30.390 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-div): div pass
2021-11-24 13:52:30.393 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-div): div fail 
                                    Expected: 4
                                    Got: 0.6666666666666666
2021-11-24 13:52:30.393 - DEBUG - (suite-nested-deftests / test-arithmetic-1-report-combined-results): Test Summary → Total: 6    Pass: 3    Fail: 3
2021-11-24 13:52:30.397 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-add): add pass
2021-11-24 13:52:30.400 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-add): add fail 
                                    Expected: 4
                                    Got: 5
2021-11-24 13:52:30.404 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-mul): mul pass
2021-11-24 13:52:30.406 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-mul): mul fail 
                                    Expected: 4
                                    Got: 6
2021-11-24 13:52:30.407 - DEBUG - (suite-nested-deftests / test-arithmetic-2-report-combined-results): Test Summary → Total: 4    Pass: 2    Fail: 2
2021-11-24 13:52:30.407 - DEBUG - (suite-nested-deftests): Test Summary → Total: 10    Pass: 5    Fail: 5
