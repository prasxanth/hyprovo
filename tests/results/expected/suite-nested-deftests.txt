2021-11-24 12:39:44.562 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results): add pass
2021-11-24 12:39:44.566 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results): sub fail 
                                    Expected: 2
                                    Got: -1
2021-11-24 12:39:44.572 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-sub): sub pass
2021-11-24 12:39:44.575 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-sub): sub fail 
                                    Expected: 4
                                    Got: -1
2021-11-24 12:39:44.581 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-div): div pass
2021-11-24 12:39:44.584 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-div): div fail 
                                    Expected: 4
                                    Got: 0.6666666666666666
2021-11-24 12:39:44.584 - DEBUG - (suite-nested-deftests / test-arithmetic-1-report-combined-results): Test Summary → Total: 6    Pass: 3    Fail: 3
2021-11-24 12:39:44.589 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-add): add pass
2021-11-24 12:39:44.592 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-add): add fail 
                                    Expected: 4
                                    Got: 5
2021-11-24 12:39:44.597 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-mul): mul pass
2021-11-24 12:39:44.601 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-mul): mul fail 
                                    Expected: 4
                                    Got: 6
2021-11-24 12:39:44.601 - DEBUG - (suite-nested-deftests / test-arithmetic-2-report-combined-results): Test Summary → Total: 4    Pass: 2    Fail: 2
2021-11-24 12:39:44.601 - DEBUG - (suite-nested-deftests): Test Summary → Total: 10    Pass: 5    Fail: 5
